packages/web_ui/src/components/InstancesPage.tsx

```diff
@@ -53,6 +59,14 @@ function CreateInstanceButton() {
 				<Form.Item name="instanceName" label="Name">
 					<Input />
 				</Form.Item>
+				<Form.Item name="instanceClone" label="Clone From">
+					<Select
+						defaultValue={-1}
+						options={[{ id: -1, name: "Default Config" }, ...props.instances.values()]
+							.map(i => ({ value: i.id, label: i.name }))
+						}
+					/>
+				</Form.Item>
```

Comment:
Some hover text explaining what is cloned would be nice - its not obvious that its only cloning the instance config and not saves for example.